sourcegraph/about
# Dev Onboarding Guide Page

### What should this PR do?

Closes #5114

### How to test?

Preview at [/guides/continuous-developer-onboarding](https://deploy-preview-5207--sourcegraph.netlify.app/guides/continuous-developer-onboarding)

1. Check page matches Figma
2. Check page matches copy
3. Check that forms work
4. Check page responsiveness and mobile views
5. Check build
